packages/project/src/android/project.ts

```diff
@@ -50,7 +50,10 @@ export class AndroidProject {
   }
 
   async setPackageName(packageName: string) {
-    this.manifest.getDocumentElement()?.setAttribute('package', packageName);
+    await Promise.all([
+      this.manifest.getDocumentElement()?.setAttribute('package', packageName),
```

Comment:
No need for the `Promise.all` here as the `setAttribute` is synchronous

Reply:
Right! Changing it right away. Thanks!